Resources/Prototypes/Entities/Structures/Piping/Atmospherics/trinary.yml

```diff
@@ -36,6 +36,7 @@
       layers:
         - sprite: Structures/Piping/Atmospherics/pipe.rsi
           state: pipeTJunction
+          rotation: -90 # fuck you!!! who cares
```

Comment:
this might not jive with the connector overlay, dunno

Reply:
the connector overlay is why i had to do that

Reply:
F